runtime/help/plugins.md

```diff
@@ -103,202 +104,202 @@ micro.Log("Hello")
-	- `OptionValueComplete`: autocomplete using names of options, and valid
-       values afterwards
-	- `NoComplete`: no autocompletion suggestions
-
-	- `TryBindKey(k, v string, overwrite bool) (bool, error)`: bind the key
-       `k` to the string `v` in the `bindings.json` file.  If `overwrite` is
-       true, this will overwrite any existing binding to key `k`. Returns true
-       if the binding was made, and a possible error (for example writing to
-       `bindings.json` can cause an error).
-
-	- `Reload()`: reload configuration files.
-
-	- `AddRuntimeFileFromMemory(filetype RTFiletype, filename, data string)`:
-       add a runtime file to the `filetype` runtime filetype, with name
-       `filename` and data `data`.
-
-	- `AddRuntimeFilesFromDirectory(plugin string, filetype RTFiletype,
-                                    directory, pattern string)`:
-       add runtime files for the given plugin with the given RTFiletype from
-       a directory within the plugin root. Only adds files that match the
-       pattern using Go's `filepath.Match`
-
-	- `AddRuntimeFile(plugin string, filetype RTFiletype, filepath string)`:
-       add a given file inside the plugin root directory as a runtime file
-       to the given RTFiletype category.
-
-	- `ListRuntimeFiles(fileType RTFiletype) []string`: returns a list of
-       names of runtime files of the given type.
-
-	- `ReadRuntimeFile(fileType RTFiletype, name string) string`: returns the
-       contents of a given runtime file.
-
-	- `NewRTFiletype() int`: creates a new RTFiletype, and returns its value.
-
-	- `RTColorscheme`: runtime files for colorschemes.
-	- `RTSyntax`: runtime files for syntax files.
-	- `RTHelp`: runtime files for help documents.
-	- `RTPlugin`: runtime files for plugin source code.
-
-	- `RegisterCommonOption(pl string, name string, defaultvalue interface{})`:
-       registers a new option with for the given plugin. The name of the
-       option will be `pl.name`, and will have the given default value. Since
-       this registers a common option, the option will be modifiable on a
-       per-buffer basis, while also having a global value (in the
-       GlobalSettings map).
-
-	- `RegisterGlobalOption(pl string, name string, defaultvalue interface{})`:
-       same as `RegisterCommonOption` but the option cannot be modified
-       locally to each buffer.
-
-	- `GetGlobalOption(name string) interface{}`: returns the value of a
-       given plugin in the `GlobalSettings` map.
-
-	- `SetGlobalOption(option, value string) error`: sets an option to a
-       given value. Same as using the `> set` command. This will parse the
-       value to the actual value type.
-
-	- `SetGlobalOptionNative(option string, value interface{}) error`: sets
-       an option to a given value, where the type of value is the actual
-       type of the value internally.
+   - `MakeCommand(name string, action func(bp *BufPane, args[]string),
+      completer buffer.Completer)`:
```

Comment:
Here, for example, the previous "long" indentation made a perfect sense.

Reply:
Once again...
> If first word started from ` it will be 3 otherwise 2

...I only followed the advice.

To be honest, the whole topic is heavily depending on personal preferences and it seems we've different opinions about readability, consistency and generalized indentation level. Currently I've no clue how to satisfy you here. I can once again jump over my own shadow, rework it again. But I've doubts that I don't receive any complaints again.

Is it worth spending that much life and free time of both of us into that PR?
Just thinking to do a break here with this PR, because it becomes to emotional.

Reply:
Ok maybe then just extract commits:

1. for deleting trailing ws and
2. for converting tabs to spaces (especially those which before default)
 
and merge?

Reply:
> ...I only followed the advice.

This particular line is clearly a special case: it is a function definition, which is split into 2 lines since its quite long, so placing `completer buffer.Completer` exactly below `name string` makes it more readable.

Or to put it more formally: this 2nd line (`completer buffer.Completer`) is not a part of the "section text", is still a continuation of the "section title", so different indentation rules apply to it.

Ditto `AddRuntimeFileFromMemory`, `JobStart` etc.

Reply:
I cooled down and overhauled it.